src/index.js

```diff
@@ -202,7 +197,6 @@ export default {
           cwd,
           stdin,
           stream: 'both',
-          timeout: this.linterTimeout,
```

Comment:
Actually, you'll want to set this to `Infinity` or the people running into the default timeout will still have issues, or set it to something higher than any lint task should take just in case (10 minutes?).